sql/2021/javascript/async_and_defer.sql

```diff
@@ -0,0 +1,25 @@
+SELECT
+  client,
+  COUNT(DISTINCT IF(async, page, NULL)) AS async,
+  COUNT(DISTINCT IF(defer, page, NULL)) AS defer,
+  COUNT(DISTINCT IF(async AND defer, page, NULL)) AS both,
+  COUNT(DISTINCT page) AS total,
+  COUNT(DISTINCT IF(async, page, NULL)) / COUNT(DISTINCT page) AS pct_async,
+  COUNT(DISTINCT IF(defer, page, NULL)) / COUNT(DISTINCT page) AS pct_defer,
+  COUNT(DISTINCT IF(async AND defer, page, NULL)) / COUNT(DISTINCT page) AS pct_both
+FROM (
+  SELECT
+    client,
+    page,
+    script,
+    REGEXP_CONTAINS(script, r'(?i)\basync\b') AS async,
+    REGEXP_CONTAINS(script, r'(?i)\bdefer\b') AS defer
+  FROM
+    `httparchive.sample_data.summary_response_bodies`  -- TODO: update to the original table name
```

Comment:
async and defer should now be available in the `_javascript` custom metric, no need to query the response bodies.

https://github.com/HTTPArchive/legacy.httparchive.org/blob/master/custom_metrics/javascript.js#L77